archive/c/c-plus-plus/primality_check.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
```

Comment:
Only issue here is the file name. Something like `prime-number.cpp` with a hyphen would be better. That way, our test suite can pick it up.